util/src/main/scala/org/ensime/util/Show.scala

```diff
@@ -0,0 +1,18 @@
+package org.ensime.util
+
+trait Show[T] {
```

Comment:
if we're going to introduce a `Show` typeclass we should at least use the one (or its pattern) from `cats` instead of creating our own.